locales/native-names.json

```diff
@@ -83,6 +87,7 @@
   "sq": "Shqip",
   "sr": "Српски",
   "sv-SE": "Svenska",
+  "sw": "sw",
```

Comment:
looks like there is no be translation yet, and therefore there is no translated native name